src/api/user/src/routes/user.js

```diff
@@ -0,0 +1,139 @@
+const { Router, logger } = require('@senecacdot/satellite');
+const { errors } = require('celebrate');
+const celebrateSchema = require('../models/celebrateSchema');
+const User = require('../models/user');
+const db = require('../services/firestore');
+
+const router = Router();
+
+// get a user with a supplied id, validated by the celebrateSchema
+// rejected if a user could not be found, otherwise user returned
+router.get('/:id', celebrateSchema.get(), async (req, res, next) => {
+  try {
+    const userRef = db.collection('users').doc(req.params.id);
+    const doc = await userRef.get();
+
+    if (!doc.exists) {
+      logger.debug(`User data (id: ${doc.id}) was requested by ${req.ip} but could not be found.`);
+      res.status(404).json({
+        msg: `User data (id: ${doc.id}) was requested but could not be found.`,
+      });
+    } else {
+      logger.debug(`User data (id: ${doc.id}) was requested by ${req.ip} and served successfully.`);
+      res.status(200).json(doc.data());
+    }
+  } catch (err) {
+    next(err);
+  }
+});
+
+// get all users
+// rejected if the user collection is empty, otherwise users returned
+router.get('/', async (req, res, next) => {
```

Comment:
This needs to be secured and also add paging support.  I filed https://github.com/Seneca-CDOT/telescope/issues/1929 to follow-up on that.